Sources/SwiftSyntaxMacrosTestSupport/Assertions.swift

```diff
@@ -129,7 +129,7 @@ public struct DiagnosticSpec {
   public let severity: DiagnosticSeverity
 
   /// If not `nil`, the text the diagnostic is expected to highlight
```

Comment:
```suggestion
  /// If not `nil`, the text fragments the diagnostic is expected to highlight
```

Reply:
fixed in `cd73ad23cc9502efcb654c2d3da4c3de95942dfd`